packages/@uppy/svelte/package.json

```diff
@@ -25,11 +28,37 @@
     "dist"
   ],
   "exports": {
-    "./dist/styles.css": "./dist/styles.css",
     ".": {
       "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
-    }
+      "svelte": "./dist/index.js",
+      "import": "./dist/index.js"
+    },
+    "./dashboard": {
+      "types": "./dist/components/Dashboard.svelte.d.ts",
+      "svelte": "./src/lib/components/Dashboard.svelte",
+      "import": "./dist/components/Dashboard.svelte.js"
+    },
```

Comment:
Are you sure this verbose-style is needed? I would think it auto resolves these with regular export statements?